services/github/github_manager.py

```diff
@@ -323,25 +323,37 @@ def create_jwt() -> str:
 @handle_exceptions(default_return_value=None, raise_on_error=False)
 def create_pull_request(body: str, title: str, base_args: BaseArgs) -> str | None:
     """https://docs.github.com/en/rest/pulls/pulls#create-a-pull-request"""
-    owner, repo, base, head, token = (
+    owner, repo, base, head, token, reviewers = (
         base_args["owner"],
         base_args["repo"],
         base_args["base_branch"],
         base_args["new_branch"],
         base_args["token"],
+        base_args["reviewers"],
     )
     response: requests.Response = requests.post(
         url=f"{GITHUB_API_URL}/repos/{owner}/{repo}/pulls",
         headers=create_headers(token=token),
-        json={"title": title, "body": body, "head": head, "base": base},
+        json={"title": title, "body": body, "head": head, "base": base, "maintainer_can_modify": True},
```

Comment:
A pure question, is this parameter related to this PR?

Reply:
I ask here https://gitauto.slack.com/archives/C06K61KM9PS/p1729140875069589?thread_ts=1729131306.347819&cid=C06K61KM9PS